strings/sillytips.txt

```diff
@@ -5,10 +5,8 @@ When a round ends nearly everything about it is lost forever, leave your salt be
 Killing the entire station isn't fun except when it is.
 You can win a pulse rifle from the arcade machine. Honest.
 Just like real life the entropy of the game can only increase with time. If things aren't on fire yet, just wait.
-Completing your objectives is good practice, but the best antagonists will strive to do more than the bare minimum to really leave an impression.
```

Comment:
we could just leave this in as a reference to contracting.

Reply:
What the fuck does this have to do with erp, it sounds way more like encouraging doing fun gimmicks rather than only doing objectives